docs/src/quickstart/hello_klio_batch.rst

```diff
@@ -1,113 +1,3 @@
-.. _hello-klio-batch:
+:orphan:
```

Comment:
(meta comment) Having both "quick start" and "user guide" as separate things could be confusing to others. As well, the quickstart has installation instructions, which could be argued should (also) be in a user guide. So I just moved the quickstart docs under the user guide.

Reply:
Also, no content was changed, just moved